detect_variants/detect_variants.cwl

```diff
@@ -17,14 +17,14 @@ inputs:
         secondaryFiles: .bai
     interval_list:
         type: File[]
-    strelka_config:
-        type: File
     dbsnp_vcf:
         type: File?
         secondaryFiles: .tbi
     cosmic_vcf:
         type: File?
         secondaryFiles: .tbi
+    strelka_exome_mode:
```

Comment:
This is currently required just as the config file was previously. I believe any input can be made optional by appending `?` to the type.

I would worry about giving people the false impression that setting this to `true` might automatically adjust cut-offs or tweak parameters for other tools.  I could possibly see hooking it up to a second tool's `--exome` flag if one happens to have one... but we might want to keep them separate even then.